src/osd/scheduler/mClockScheduler.cc

```diff
@@ -39,6 +42,18 @@ mClockScheduler::mClockScheduler(CephContext *cct) :
     cct->_conf.get_val<double>("osd_mclock_scheduler_anticipation_timeout"))
 {
   cct->_conf.add_observer(this);
+  // Set per op shard iops limits
+  ceph_assert(num_shards > 0);
+  client_iops_limit =
+    cct->_conf.get_val<double>("osd_mclock_max_client_iops");
+  client_iops_limit /= num_shards;
```

Comment:
it would be handy to be able to change this at runtime

Reply:
Replaced the above with osd_max_capacity_iops_(hdd, ssd). Will update the PR with additional changes to handle_conf_change() for all the tracked config options.